.github/PULL_REQUEST_TEMPLATE/bep_template.yml

```diff
@@ -0,0 +1,29 @@
+---
+-   number:
+    title: # full title
+    display: # short title for display
+    google_doc:
+    pull_request: # pull request for the BEP on the BIDS specification repo
+    html_preview: # link to the html render of the pull request
+                  #  for the BEP on the BIDS specification repo
+    leads:
+  # MUST match given-names and family-names
+  #  from the bids specification citation.cff
+  # Update the list of contributors via the BIDS specification
+  #  if necessary.
+  # See: https://github.com/bids-standard/bids-specification/wiki/Recent-Contributors#adding-yourself-as-a-contributor
+    -   given-names: ' '
+        family-names: ' '
+    bids_maintainers:
+    -   given-names: ' '
+        family-names: ' '
+    status: # updates
+    -   '  ' # add updates
+    -   '  ' # with items
+    blocking: # potential blockers for this BEP
+    -   ' ' # add blocker
+    -   ' ' # with items
+    communication_channel: # mattermost, matrix, slack, discord...
+    google_doc_created:
+    pull_request_created:
+    pull_request_merged:
```

Comment:
could/should we add specific examples/templates here in `# comments` since otherwise unclear what to specify here?